demos/cache_sidechannel.cc

```diff
@@ -51,7 +51,7 @@ void CacheSideChannel::FlushOracle() const {
   }
 }
 
-std::pair<bool, char> CacheSideChannel::RecomputeScores(char safe_offset_char) {
+std::pair<bool, char> CacheSideChannel::RecomputeScores(size_t safe_offset) {
```

Comment:
Seems like changing this to `size_t` required adding a cast at every call site. Was it nicer as `char`?